speed/.eslintrc

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
The files should be named `.eslintrc.json`, not `.eslintrc`. The latter is deprecated.